pkg/apis/core/types_shoot.go

```diff
@@ -536,6 +536,41 @@ type KubeAPIServerConfig struct {
 	EnableAnonymousAuthentication *bool
 	// EventTTL controls the amount of time to retain events.
 	EventTTL *metav1.Duration
+	// AccessControl provides access control configurations for the API server.
+	AccessControl *AccessControl
+}
+
+// AccessControl provides authorization mechanisms for Shoot-API-Server.
```

Comment:
Nit (to keep this more inline with other comments):
```suggestion
// AccessControl provides authorization mechanisms for the kube-apiserver.
```